package.json

```diff
@@ -46,24 +46,24 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@cucumber/cucumber": "^8.3.0",
-    "@cucumber/cucumber-expressions": "^15.2.0",
+    "@cucumber/cucumber": "^8.5.1",
+    "@cucumber/cucumber-expressions": "^16.0.0",
     "@cucumber/gherkin": "^24.0.0",
     "@cucumber/gherkin-streams": "^5.0.1",
     "@cucumber/message-streams": "^4.0.1",
-    "@cucumber/messages": "^19.0.0"
+    "@cucumber/messages": "^19.1.2"
   },
   "peerDependencies": {
-    "@cucumber/cucumber": "^8.0.0",
-    "@cucumber/cucumber-expressions": "^15.0.0",
-    "testcafe": "~1.18.0"
+    "@cucumber/cucumber": "^8.5.1",
+    "@cucumber/cucumber-expressions": "^16.0.0",
```

Comment:
This would have been necessary if we want to force the users to switch to version 16, but at the moment I'd rather keep it to version 15 so you can revert back to `^15.0.0`